app/Console/Commands/DatabaseWipeCommand.php

```diff
@@ -47,51 +40,49 @@ public function __construct()
      */
     public function fire()
     {
-        if (! $this->confirm('Did you transfer the winners data to the winners table? [y|n]')) {
-            return $this->error('Please run artisan transfer:winners command!');
+        if (! $this->confirm('If this is Footlocker External, is the winner gallery finished? [y|n]')) {
+            return $this->error('Please award scholarships before running!');
         }
 
-        if (! $this->confirm('Did you run a dump and back up the current database? [y|n]')) {
-            return $this->error('Please back up the current database before proceeding!');
+        if (! $this->confirm('Did you take a snapshot of the current database? [y|n]')) {
+            return $this->error('Please take a snapshot of the current database before proceeding!');
         }
 
-        // Get the admins from the Users table.
-        $admins = DB::select(DB::raw('SELECT *
-						                      FROM users u
-						                      INNER JOIN role_user ru
-						                      WHERE ru.id = u.id;'));
-
-        DB::statement('SET FOREIGN_KEY_CHECKS=0;');
-
         // Clear out database tables.
-        Application::truncate();
-        Nomination::truncate();
-        Profile::truncate();
-        Race::truncate();
-        Rating::truncate();
-        Recommendation::truncate();
-        RecommendationToken::truncate();
-        User::truncate();
+        DB::table('applications')->delete();
+        DB::table('nominations')->delete();
+        DB::table('profiles')->delete();
+        DB::table('races')->delete();
+        DB::table('ratings')->delete();
+        DB::table('recommendations')->delete();
+        DB::table('recommendation_tokens')->delete();
         if ((Schema::hasTable('failed_jobs'))) {
```

Comment:
This check can be removed, since every instance has this table now (created [with this migration](https://github.com/DoSomething/longshot/blob/master/database/migrations/2018_09_11_170552_create_failed_jobs_table.php)).

Reply:
OOOO great eye @DFurnes !!!